store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1261,6 +1264,11 @@ public void run() {
         // DefaultMessageStore.this.cleanExpiredConsumerQueue();
         // }
         // }, 1, 1, TimeUnit.HOURS);
+        this.diskCheckScheduledExecutorService.scheduleAtFixedRate(new Runnable() {
```

Comment:
the code coverage is decrased when I add shutdown code... should I fix it or just ignore it？Does this affect merge of this pr？

Reply:
yes, due to the code coverage decreased by more than 1%, so it would be nice if you can add some unit tests, especially for the isSpaceFull()  method.